dotgov-domains/current-federal.csv

```diff
@@ -368,10 +368,10 @@ GLOBALENTRY.GOV,Federal Agency - Executive,Department of Homeland Security,U.S.
 HOMELANDSECURITY.GOV,Federal Agency - Executive,Department of Homeland Security,Department of Homeland Security,Washington,DC
 ICE.GOV,Federal Agency - Executive,Department of Homeland Security,U.S. Immigration and Customs Enforcement,Washington,DC
 LISTO.GOV,Federal Agency - Executive,Department of Homeland Security,Office of Public Affairs,Washington,DC
+NIEM.GOV,Federal Agency - Executive,Department of Homeland Security,"Department of Homeland Security, Office of the Chief Information Officer",Washington ,DC

```

Comment:
Would it make sense to ask DHS to use an `Office` value without repeating the Agency name? Other DHS domain's office fields are listed without it.

Reply:
Also, there's an extra space at the end of `Washington ` there. It'd be worth cleaning that up to avoid errors for people aggregating on naive string matches.

Reply:
The .gov registrar should recommend what agencies should use in the Org field, yeah...

Thanks for catching the niem.gov error -- that will be fixed in the next push.